packages/dht/test/kademlia-simulation/SortedContactList.ts

```diff
@@ -0,0 +1,104 @@
+import KBucket from 'k-bucket'
+import { Contact } from './Contact'
+
+class ContactWrapper {
+    public contacted = false
+    public active = false
+    constructor(public contact: Contact) {
+    }
+}
+
+export class SortedContactList {
```

Comment:
Get rid of old code in, remove duplication in `kademlia-simulation`

Reply:
Removed duplicate SortedContactList